.ci/release.sh

```diff
@@ -5,6 +5,28 @@ set -eou pipefail
 DOWNLOAD_BASE="$1"
 VERSION="$2"
 
+install_homebrew() {
+  # do an alternative install of homebrew to ensure we have access to
+  # perform brew operations
+  # https://docs.brew.sh/Installation#untar-anywhere
+  # note that homebrew needs to be installed somewhere with a shortish
+  # path to avoid issues relinking dynamic libraries
+  # https://github.com/Homebrew/brew/issues/4979
+  rm -rf "/tmp/b"
```

Comment:
Do you need to test for existence of the directory before removing it to handle the first-run use case without erroring out?

Also, would it make sense to just call `remove_homebrew` here to avoid the duplication?

Reply:
> Do you need to test for existence of the directory before removing it to handle the first-run use case without erroring out?

No, the `-f` flag on `rm` ignores non-existent files.

> Also, would it make sense to just call remove_homebrew here to avoid the duplication?

I think so, will do.

Reply:
Done in https://github.com/elastic/homebrew-tap/pull/78/commits/90c7e433acc9677e1b02544a2003dc5f3da619b7